### PR TITLE
Google Lighthouse and W3C Validation Improvements

### PR DESCRIPTION
**Pull request by t51r**
Implemented Google Lighthouse and W3C optimizations, including:
- Added HTML `lang` attribute, `charset` meta tag, and tweaked `viewport` meta tag
- Removed unnecessary style tag `type` attribute
- Added missing image `alt`, `width`, and `height` attributes
- Resized and compressed images
- Switched to hierarchical headers (H1 / H2) and adjusted CSS accordingly
- Added `aria-label` attributes to several items
- Fixed W3C validation issues with several missing or unnecessary element closing tags (most paragraphs and divs)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -45,16 +47,16 @@
 </head>
 
 <body>
-<img id="logo" src="respect.png" />
+<img id="logo" src="respect-small.png" alt="Residency Inspection logo." width="350" height="134" />
```
Comment (didtheyvotein) [inline about.html:50]: "Respect" is the company name; "Residency Inspection" is just text for the logo. (So, alt should be "Respect logo")

File: about.html [hunk 2]
```diff
@@ -70,27 +72,27 @@
 
 <div id="content" class="tight">
 
-<h>Passionate about fairness.</h>
+<h1>Passionate about fairness.</h1>
```
Comment (didtheyvotein) [inline about.html:75]: All should be either h2 or h1 (headers are on same level -- "Passionate about fairness") isn't a title for the entire page

File: banks.html [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
-<html>
+<html lang="en">
 
 <head>
-<title>Respect</title>
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Residential & Business Address Verification Services for Banks</title>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+<meta name="description" content="Respect services offers residential and business address verification services that can help ensure the integrity of your customer identification program." />
```
Comment (didtheyvotein) [inline banks.html:8]: We may wish to stick with the more concrete description that's on the Overview page. ("Address Verification Service" has a very specific meaning to financial institutions, and this text doesn't speak to schools.) But, we're still thinking of the least-clumsy way to describe this.

File: banks.html [hunk 4]
```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
-<title>Respect</title>
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Residential & Business Address Verification Services for Banks</title>
```
Comment (didtheyvotein) [inline banks.html:5]: We do need a description of the service in the <title> - but AVS can't be it. (And company name "Respect" should probably stay.)

File: walkthrough.html [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
-<html>
+<html lang="en">
 
 <head>
-<title>Respect</title>
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Address Verification Service Walkthrough</title>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+<meta name="description" content="Discover the Respect Services process for submitting address verification requests, accessing and removing submitted data." />
```
Comment (didtheyvotein) [inline walkthrough.html:8]: See prior notes about "AVS"

File: respect.css [hunk 6]
```diff
@@ -250,7 +254,7 @@ em {
 }
 
 button { 
-  color: white;
+  color: var(--purple);
```
Comment (didtheyvotein) [inline respect.css:257]: Is there a reason for this?

File: walkthrough.html [hunk 7]
```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
-<title>Respect</title>
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Address Verification Service Walkthrough</title>
```
Comment (didtheyvotein) [inline walkthrough.html:5]: AVS

**Review by didtheyvotein — CHANGES_REQUESTED** [review]
Overall I think this is a lot of good cleanup & optimization. 

- We need to decide on a phrase besides "Address Verification Service". 
- The `h1` / `h2` use suggests hierarchy where there isn't any
- I prefer the white button text, but I can be convinced otherwise based on what others think

File: respect.css (hunk 6 above)
Comment (t51r) [reply respect.css:257]: White on that background color has poor color contrast, and Lighthouse calls it out as an accessibility issue.

File: about.html (hunk 2 above)
Comment (t51r) [reply about.html:75]: By HTML spec, and from an SEO and accessibility standpoint, there should really be a single H1 per page. All other headers should be H2-H6. Generally I don't go beyond H2-3.

File: about.html (hunk 1 above)
Comment (t51r) [reply about.html]: Changed all logo alt text to "Respect logo."

File: banks.html (hunk 4 above)
Comment (t51r) [reply banks.html:5]: I'd typically recommend for title tags that we rear brand with a "pipe" for consistency sake:
e.g. "Residential & Business Blah Blah | Respect"

File: banks.html (hunk 3 above)
Comment (t51r) [reply banks.html:8]: Let me know what the preferred text is and I'll change it. Generally for meta descriptions I try to stick to ~156 characters or less.

File: respect.css (hunk 6 above)
Comment (didtheyvotein) [reply respect.css:257]: OK - let's try the purple.

File: banks.html (hunk 4 above)
Comment (didtheyvotein) [reply banks.html:5]: I like pipes.

File: about.html (hunk 2 above)
Comment (didtheyvotein) [reply about.html:75]: Perhaps we can keep them as `h2` on these small pages? (And then use `h1` on our content pages which have meaningful titles). It's not a big deal either way.

File: banks.html (hunk 3 above)
Comment (didtheyvotein) [reply banks.html:8]: Honestly a hard question :smile: "Address validation" is also taken (by USPS, UPS, etc.)

"Inspection" kinda has the right ring (not as intense as "investigation", but suggests more manual effort and fastidiousness than "processing" or "validation"). But it's not clear enough for a `meta` tag. 

Any thoughts?

Comment (didtheyvotein) [reply banks.html:8]: For the `title`, how about some formatting of "Respect: find address fraud while preserving privacy"?

For the `meta` tag, how about "Respect processes address databases (without attached names) to ensure that students belong to school districts and bank accounts have legitimate records." (We'll probably need to change this.)

**Review by didtheyvotein — APPROVED** [review]

**Pull request merged**